website/docusaurus.config.js

```diff
@@ -54,7 +54,7 @@ module.exports = {
         announcementBar: {
             id: 'supportus',
             content:
-              '⭐️  &nbsp; If you like WebdriverIO, give it a star on <a target="_blank" rel="noopener noreferrer" href="https://github.com/${organizationName}/${projectName}">GitHub</a>! ⭐️',
+              '⭐️  &nbsp; If you like WebdriverIO, give it a star on <a target="_blank" rel="noopener noreferrer" href="https://github.com/webdriverio/webdriverio">GitHub</a>! ⭐️',
```

Comment:
Good catch!

The variables are defined at the top of this file, however the string wasn't a string template. Instead do the following.
```suggestion
              `⭐️  &nbsp; If you like WebdriverIO, give it a star on <a target="_blank" rel="noopener noreferrer" href="https://github.com/${organizationName}/${projectName}">GitHub</a>! ⭐️`,
```

Reply:
Done as suggested. I should have thought about this. 👍

Reply:
No worries, thanks for fixing this. LGTM 👍